### PR TITLE
[chore] test_generate_payload_fixtures: pin object IDs 

### PR DESCRIPTION
**Pull request by jevonyeoh**
## Description
Pin object IDs to fixed ones so that all the json's don't get regenerated with new IDs everytime we run the task command to generate fixtures.

There are still a few more IDs that need to be pinned, but this should cover most of them for now.

## Type of Change

Label this pull request to place it under the correct category in Release Notes:

|        **Pull Request Label**         | **Category in Release Notes** |
|:-------------------------------------:|:-----------------------------:|
|       `enhancement`, `feature`        |          🚀 Features          |
| `bug`, `refactoring`, `bugfix`, `fix` |    🔧 Fixes & Refactoring     |
|       `build`, `ci`, `testing`        |    📦 Build System & CI/CD    |
|              `breaking`               |      💥 Breaking Changes      |
|            `documentation`            |       📝 Documentation        |
|            `dependencies`             |    ⬆️ Dependencies updates    |

## Checklist

- [ ] I have read the [`CODE_OF_CONDUCT.md`](https://github.com/featurebyte/featurebyte/blob/main/CODE_OF_CONDUCT.md) and [`CONTRIBUTING.md`](https://github.com/featurebyte/featurebyte/blob/main/CONTRIBUTING.md) guides.
- [ ] I have written tests for the changes made.
- [ ] I have written docstrings in [NumpyDoc format](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard)
- [ ] I have labeled my Pull Request correctly

### CODE REVIEW

File: tests/unit/api/test_target_table.py [hunk 1]
```diff
@@ -27,6 +27,5 @@ def assert_info_dict(self, info_dict: Dict[str, Any]) -> None:
             },
             "created_at": info_dict["created_at"],
             "updated_at": None,
-            "catalog_description": None,
```
Comment (jevonyeoh) [inline tests/unit/api/test_target_table.py:30]: hm i'm not sure why this change is required even though i'm rebased on master 🤔

File: tests/unit/test_generate_payload_fixtures.py [hunk 2]
```diff
@@ -35,6 +38,15 @@ def request_payload_dir_fixture():
     return "tests/fixtures/request_payloads"
 
 
+def replace_obj_id(obj: Any, obj_id: ObjectId) -> Any:
+    """
+    Helper function to replace the object ID of the type
+    """
+    params = obj.dict()
+    params["_id"] = obj_id
```
Comment (gee-senbong) [inline tests/unit/test_generate_payload_fixtures.py:46]: If we replace `obj.dict()` to `obj.dict(by_alias=True)`, it should fix the issue without introducing this method.

**Review by gee-senbong — APPROVED** [review]
